README.md

```diff
@@ -1,372 +1,221 @@
 
-**Live example, including server rendering + redux:** http://www.reactrails.com/
++ [Redux](https://github.com/rackt/redux)
++ [Webpack dev server](https://webpack.github.io/docs/webpack-dev-server.html) with [hot module replacement](https://webpack.github.io/docs/hot-module-replacement-with-webpack.html)
++ [Webpack optimization functionality](https://github.com/webpack/docs/wiki/optimization)
++ **(Coming Soon)** [React Router](https://github.com/rackt/react-router)
 
-Sponsored by [ShakaCode.com](http://www.shakacode.com/)
+See the [react-webpack-rails-tutorial](https://github.com/justin808/react-webpack-rails-tutorial/) for an example of a live implementation and code.
 
-See [Action Plan for v1.0](https://github.com/shakacode/react_on_rails/issues/1). We're ready v1.0!
+### Including your React Component in your Rails Views
+Please see [Getting Started](#getting-started) for how to set up your Rails project for React on Rails if you have not already done so.
 
-Feedback and pull-requests encouraged! Thanks in advance! We've got a private slack channel to discuss react + webpack + rails. [Email us for an invite contact@shakacode.com](mailto: contact@shakacode.com).
++ *Normal Mode (JavaScript is rendered on client):*
 
-Supports:
+  ```erb
+  <%= react_component("HelloWorldApp", { @some_props }) %>
+  ```
++ *Server-Side Rendering:*
+
+  ```erb
+  <%= react_component("HelloWorldApp", { @some_props }, prerender: true) %>
+  ```
+
++ The `component_name` parameter here would be a string matching the name you used when globally exposing your React component.
++ `@some_props` can be either a hash or JSON string or {} if you have no props. This will make the data available in your component:
 
-1. Rails
-2. Webpack
-3. React, both v0.14 and v0.13.
-4. Redux
-5. Turbolinks
-6. Server side rendering with fragment caching
-7. react-router for client side rendering (and server side very soon)
+  ```javascript
+    this.props.name
+  ```
 
-## OPEN ISSUES
-1. Almost all the open issues are nice to haves like more tests.
-2. If you want to work on any of the open issues, please comment on the issue. My team is mentoring anybody that's trying to help with the issues.
-3. Longer term, we hope to put in many conveniences into this gem, in terms of Webpack + Rails integration. We're open to suggestions.
+### Client-Side Rendering vs. Server-Side Rendering
+In most cases, you should use the provided helper method to render the React component from your Rails views with `prerender: false` (default behavior). In some cases, such as when SEO is vital or many users will not have JavaScript enabled, you can pass the `--server-rendering` option to the generator to configure your application for server-side rendering. Your JavaScript can then be first rendered on the server and passed to the client as HTML.
```

Comment:
"many users will not have JavaScript enabled"

not really worth optimizing for any more

instead you might do for client performance, especially if you can fragment cache the pre-rendered html, thus the pre-rendered page will show more quickly than if done via client rendering.